### PR TITLE
Propose idea for vitess term 3 of 2024

### PR DESCRIPTION
**Pull request by frouioui**
This PR propose a project for Vitess. cc @mattlord

https://github.com/vitessio/website/issues/1798

### CODE REVIEW

**Comment by aliok** [comment]
Thanks @frouioui 

Can @mattlord give us a 👍 here?

File: programs/lfx-mentorship/2024/03-Sep-Nov/project_ideas.md [hunk 1]
```diff
@@ -365,3 +365,16 @@ Expected Outcome: Gadget developers have a way to run unit tests in different ke
...
 - Upstream Issue: https://github.com/kubearmor/KubeArmor/issues/1815
+
+
+### Vitess
+
+#### Add new getting started examples
+
+- Description: Vitess is a cloud-native database, while managing Vitess can be complex, the list of our getting started guide and code examples is not very exhaustive. We would like to have a mentee work on growing the list of code examples and guide to help new users acquire Vitess.
+- Expected Outcome: By the end of the term, the mentee will have a deeper knowledge of Vitess and shipped at least one guide in every area of Vitess.
+- Recommended Skills: Go, DDBS, Communication
```
Comment (mattlord) [inline programs/lfx-mentorship/2024/03-Sep-Nov/project_ideas.md:376]: I'm not sure how well known DDBS is, IMO it's better to use distributed databases

**Comment by mattlord** [comment]
> Can @mattlord give us a 👍 here?

👍 

Thanks @frouioui and @aliok !

File: programs/lfx-mentorship/2024/03-Sep-Nov/project_ideas.md (hunk 1 above)
Comment (frouioui) [reply programs/lfx-mentorship/2024/03-Sep-Nov/project_ideas.md]: fixed via ff98932

**Review by nate-double-u — APPROVED** [review]
Thanks for this @frouioui & @mattlord!

**Pull request merged**